### ISSUE TITLE
Alpine Linux: Broken on SAM Linux AMD64 Binary 0.2.11+

### ISSUE THREAD

**Issue by zippadd** [awslabs/aws-sam-cli#248]
Executing SAM 0.2.11+ binary (from the .tar.gz package) in an Alpine Linux docker container gives the following errors:

```
Error relocating /usr/local/bin/sam: __vfprintf_chk: symbol not found
Error relocating /usr/local/bin/sam: __fprintf_chk: symbol not found
The command '/bin/sh -c sam' returned a non-zero code: 127
```

Confirmed that 0.2.10 works, giving the expected usage information.

Based on research, it appears that SAM is linking to glibc. Alpine only supports MUSL, so the needed symbols aren't present.

Here's my docker container (with link to the Dockerfile on Github) for reference/reproduction purposes: https://hub.docker.com/r/zippadd/aws-serverless-node-docker-alpine/

**Comment by zippadd** [comment]
Interestingly enough, this was fixed in 0.2.10 and broken again in 0.2.11.

**Comment by zippadd** [comment]
0.3.0 uses Python (and PIP) now instead of Go, so I'll close this for now and see if issues crop up.
